### PR TITLE
update Homepage submission grading

### PR DESCRIPTION
**Pull request by curiouskiwi**

### CODE REVIEW

File: homepage/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import check50
+
+@check50.check()
+def submitted():
+    """Homepage submitted"""
+
```
Comment (dmalan) [inline homepage/__init__.py:6]: Can't have an empty function body. Probably needs `pass`.

**Review by dmalan — CHANGES_REQUESTED** [review]

File: homepage/__init__.py (hunk 1 above)
Comment (curiouskiwi) [reply homepage/__init__.py:6]: I copied that from 2019/x  so assumed it was still the same.  Yes, @kzidane please review.

Comment (Jelleas) [reply homepage/__init__.py:6]: `pass` isn't needed here because the docstring in Python is actual code that also happens to be used as a comment.

Comment (dmalan) [reply homepage/__init__.py:6]: 🤯

Comment (curiouskiwi) [reply homepage/__init__.py:6]: okay, I merged.

**Pull request merged**